[Failed Test] policies_spec.js

```
Running:  plugins/index-management-dashboards-plugin/policies_spec.js                    (8 of 17)


  Policies
    can be created
      ✓ successfully (20543ms)
      ✓ with an alias action using the visual editor (11212ms)
    can be edited
      ✓ successfully (11403ms)
      1) with more aliases
    can be deleted
      ✓ successfully (6512ms)
    can be searched
      ✓ successfully (9114ms)
    can be viewed
      ✓ successfully (3509ms)


  6 passing (2m)
  1 failing

  1) Policies
       can be edited
         with more aliases:
     AssertionError: Timed out retrying after 60000ms: Expected to find content: 'alias3' within the element: <div.euiComboBox> but never did.
      at eval (http://localhost:5601/__cypress/tests?p=cypress/integration/plugins/index-management-dashboards-plugin/policies_spec.js:487:63)
  at Array.forEach (<anonymous>)
      at Context.eval (http://localhost:5601/__cypress/tests?p=cypress/integration/plugins/index-management-dashboards-plugin/policies_spec.js:486:25)




  (Results)

  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Tests:        7                                                                                │
  │ Passing:      6                                                                                │
  │ Failing:      1                                                                                │
  │ Pending:      0                                                                                │
  │ Skipped:      0                                                                                │
  │ Screenshots:  1                                                                                │
  │ Video:        true                                                                             │
  │ Duration:     2 minutes, 11 seconds                                                            │
  │ Spec Ran:     plugins/index-management-dashboards-plugin/policies_spec.js                      │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
```

https://github.com/opensearch-project/index-management-dashboards-plugin/actions/runs/6117128934/job/16603304492
https://github.com/opensearch-project/index-management-dashboards-plugin/actions/runs/6278254496/job/17051607555
https://github.com/opensearch-project/index-management-dashboards-plugin/actions/runs/6383883900/job/17325398343
https://github.com/opensearch-project/index-management-dashboards-plugin/actions/runs/6713125910/job/18244083343
https://github.com/opensearch-project/index-management-dashboards-plugin/actions/runs/6714280704/job/18247266546?pr=909

--- a/cypress/integration/plugins/index-management-dashboards-plugin/policies_spec.js
+++ b/cypress/integration/plugins/index-management-dashboards-plugin/policies_spec.js
@@ -215,7 +215,7 @@ describe("Policies", () => {
       // Click on the test alias to navigate to the details page
       const testInputs = {
         add: ["alias4", "alias6"],
-        remove: ["alias1", "alias2", "alias3", "alias7"],
+        remove: ["alias1", "alias2", "alias7"],
       };
       /* Edit the policy */
 

